src/ServiceInsight/SequenceDiagram/SequenceDiagramView.xaml

```diff
@@ -114,7 +114,7 @@
                                         <Label Grid.Row="0" Grid.Column="1" Content="{Binding FullName}" Style="{StaticResource TooltipLabelStyle}" />
                                         <Label Grid.Row="1" Grid.Column="0" Content="NSB Version" Style="{StaticResource TooltipLabelStyle}" />
                                         <Label Grid.Row="1" Grid.Column="1" Content="{Binding Version}" Style="{StaticResource TooltipLabelStyle}" />
-                                        <Label Grid.Row="2" Grid.Column="0" Content="Host" Style="{StaticResource TooltipLabelStyle}" />
+                                        <Label Grid.Row="2" Grid.Column="0" Content="HostNames" Style="{StaticResource TooltipLabelStyle}" />
```

Comment:
Why this change? wouldn't in this diagram we would want to see the exact host that handled the message ?